conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureHandler.java

```diff
@@ -51,6 +53,22 @@ private ConjureHandler(
             List<Endpoint> endpoints) {
         this.routingHandler = Handlers.routing().setFallbackHandler(fallback);
         endpoints.forEach(this::register);
+        registerOptionsEndpoints(endpoints);
+    }
+
+    private void registerOptionsEndpoints(List<Endpoint> endpoints) {
+        ImmutableSetMultimap<String, HttpString> pathToMethod = endpoints.stream()
+                .filter(endpoint -> !Methods.OPTIONS.equals(endpoint.method()))
```

Comment:
we probably ought to error if we ever find any of these?

Reply:
Removed this filter in favor of validation on the builder to prevent any methods that aren't defined by conjure.